specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/examples/Labels_CreateOrUpdate.json

```diff
@@ -0,0 +1,117 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "labels-rg",
+    "containerAppName": "myapp",
+    "labelsName": "current",
+    "api-version": "2023-11-02-preview",
+    "labelsEnvelope": {
+      "properties": {
+        "labels": {
+          "production": {
+            "revisionName": "myapp-ab1234",
+            "trafficWeight": 100
+          },
+          "canary": {
+            "revisionName": "myapp-cd5678",
+            "trafficWeight": 0
+          }
+        }      
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/labels-rg/providers/Microsoft.App/containerApps/myapp/labels/current",
+        "name": "current",
+        "type": "Microsoft.App/containerapps/labels",
+        "properties": {
+          "operationState": "Succeeded",
+          "labels": {
+            "production": {
+              "revisionName": "myapp-ab1234",
+              "trafficWeight": 100,
+              "template": {
```

Comment:
What happens if the template object gets roundtripped on a subsequent PUT request to update labels?

Reply:
@ehamai - the template element is read-only. It will be ignored if client roundtrips it.